homeassistant/components/fritz/strings.json

```
@@ -160,11 +160,11 @@
       "description": "Sets a new password for the guest Wi-Fi. The password must be between 8 and 63 characters long. If no additional parameter is set, the password will be auto-generated with a length of 12 characters.",
       "fields": {
         "device_id": {
-          "name": "Fritz!Box Device",
-          "description": "Select the Fritz!Box to check."
+          "name": "[%key:component::fritz::services::reconnect::fields::device_id::name%]",
+          "description": "[%key:component::fritz::services::cleanup::fields::device_id::description%]"
```

this was a bug before (_copy&paste issue_)
```suggestion
          "description": "Select the Fritz!Box to configure."
```